Homebrew/Archetypes/Ranger/Ranger (Wasteland Wanderer).xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<compendium version="5">
+	<subclass baseclass="Ranger" HB="HB">
+		<name>Ranger (Wasteland Wanderer)</name>
+		<autolevel level="3">
+			<feature optional="YES">
+				<name>Ranger Conclave: Wasteland Wanderer Conclave</name>
+				<text>Some worlds thrive, with large swathes of the land covered with  ora and fauna that can be both beautiful and dangerous while other lands, or other worlds, become desolate wastelands through cataclysmic occurrences either natural or manufactured.</text>
+				<text>Apocalyptic events turn the land into hazardous voids more dangerous than any monster. Only the hardiest life forms survive, and even they are often mutated into hideous reflections of their natural selves.</text>
+				<text>The Sea of Silt in Dark Sun's Athas, the Mournland of Eberron, and the spellplagued areas of Faerûn are just some examples in lore.</text>
+				<text>Forged in the harsh kiln of apocalyptic survival are the Wasteland Wanderer. These rangers of ruin rely on the skills and knowledge they have gathered by surviving the terrible conditions of their world— and often helping others survive as well.</text>
+			</feature>
+			<feature optional="YES">
+				<name>Wasteland Wanderer Conclave: Wasteland Wanderer Magic</name>
+				<text>Starting at 3rd level, you learn an additional spell with you reach certain levels in this class, as shown in the Wasteland Wanderer Spells table. The spell counts as a ranger spell for you, but it doesn't count against the number of ranger spells you know.</text>
+				<text />
+				<text>3rd-purify food and drink</text>
+				<text>5th-aid</text>
+				<text>9th-mass healing word</text>
+				<text>13th-death ward</text>
+				<text>17th-mass cure wounds</text>
+			</feature>
+			<feature optional="YES">
+				<name>Wasteland Wanderer Conclave: Vigilant Guardian</name>
+				<text>At 3rd level, you become harder to ambush and can't be surprised while within your favored terrain. If a member of your party is surprised at the start of combat, you have advantage on initiative rolls.</text>
+				<text>	Additionally, you can use your bonus action to make a Wisdom (Perception) or Intelligence (Investigation) check to notice hidden enemies. If you notice a hidden creature, you can immediately move up to half your speed and make one additional weapon attack against that target, as part of that same bonus action. When you do, you have advantage on the attack and you deal an additional 2d8 damage on a hit.</text>
+			</feature>
+		</autolevel>
+		<autolevel level="7">
+			<feature optional="YES">
+				<name>Wasteland Wanderer Conclave: Trail of Trouble</name>
+				<text>Starting at 7th level, you have a +5 bonus to your passive Wisdom (Perception) and passive Intelligence (Investigation) scores. You gain this bene t even while traveling at a normal pace, instead of a slow pace, while you are in your favored terrain.</text>
```

Comment:
```suggestion
				<text>Starting at 7th level, you have a +5 bonus to your passive Wisdom (Perception) and passive Intelligence (Investigation) scores. You gain this benefit even while traveling at a normal pace, instead of a slow pace, while you are in your favored terrain.</text>
```